hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -68,14 +70,17 @@ class FetcherConfig:
     s3_bucket: str
     s3_te_data_folder: str
     collab_config_table: str
+    data_store_table: str
 
     @classmethod
-    @lru_cache(maxsize=1)  # probably overkill, but at least it's consistent
```

Comment:
What's this for?

Reply:
https://github.com/facebook/ThreatExchange/pull/503#discussion_r615995597